### ISSUE TITLE
Accessibility needs to be looked at

### ISSUE THREAD

**Issue by LukaszJaro** [michelegiorgi/formality#48]
Integration with Gutenberg is very nice and conditional fields!

The accessibility issues make it unusable.

- When pressing enter on empty input field it jumps to next input instead of showing a list of errors on top of the form
- On input error, focus should jump to list of errors above the form
- Also screen readers are unable to read the errors because of focus change to next input
- Error not being read to screen readers(errors not associated with input)

Here's a example of a accessible form:

https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html

![image](https://user-images.githubusercontent.com/10998875/156035694-b84c68f2-fa4c-4148-9694-ef352e499ee8.png)

**Comment by alfianridwan** [comment]
+1 to accessibility. User also can't go back to the previous input using Shift + Tab (it acts like a Tab, moving forward), though Tab works fine.
